crypto/src/test/kotlin/web5/sdk/crypto/AwsKeyManagerTest.kt

```diff
@@ -0,0 +1,70 @@
+package web5.sdk.crypto
+
+import com.amazonaws.AmazonServiceException
+import com.amazonaws.auth.AWSStaticCredentialsProvider
+import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.services.kms.AWSKMSClient
+import com.nimbusds.jose.JWSAlgorithm
+import com.nimbusds.jose.jwk.KeyType
+import com.nimbusds.jose.jwk.KeyUse
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import kotlin.test.assertEquals
+
+class AwsKeyManagerTest {
+
+  val signingInput = "The Magic Words are Squeamish Ossifrage".toByteArray()
+  val awsKeyManager = AwsKeyManager()
+
+//  @Test
```

Comment:
intended to be commented out?

Reply:
Needs AWS creds, so won't work in GHA. I tried a few different ways of disabling by annotation, and gave up in frustration.

Reply:
is `@Ignore` or `@Disabled` not working because we haven't pulled junit in as a dependency?

Reply:
Both annotations are working locally for me. Trying to match versions of Gradle and Java to repro the problem, but not having any luck

Reply:
We have
```
tasks.test {
  useJUnitPlatform()
}
```

Reply:
tested by adding `testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")` as a dependency then tried using `@Disabled` and `@EnabledIfEnvironmentVariable(named = "AWS", matches = "something")`. both worked

<img width="1082" alt="image" src="https://github.com/TBD54566975/web5-kt/assets/4887440/f1a397e5-2fe5-4609-9848-02a5a1e34b54">

Reply:
Resolved it. I made a silly mistake by instantiating AwsKeyManager outside the test methods